packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyState.twig

```diff
@@ -28,56 +26,60 @@
     {{ styleProp(_styleProps) }}
     {{ classProp(_classNames) }}
 >
-    <div class="{{ _wrapperClassName }}">
-        {% if _iconName %}
-            <div class="{{ _iconWrapperClassName }}">
-                <Icon name="{{ _iconName }}"/>
-            </div>
-        {% endif %}
-        <div>
-            <Heading
-                elementType="{{ _headingElementType }}"
-                size="xsmall"
-                data-test-empty-tile="{{ _headingElementType }}"
-                UNSTABLE_className="{{ _headingClassName }}"
-            >
-                {{ _title }}
-            </Heading>
-            <Text color="secondary">{{ _description | raw }}</Text>
-        </div>
-    </div>
-    {% if block('content') is not empty %}
+    {% if _heading or _description or _imageSlot %}
         <div class="{{ _contentClassName }}">
-            {% block content %}{% endblock %}
+            {% if _imageSlot is not empty %}
+                {{ _imageSlot | raw }}
+            {% endif %}
+            {% if _heading %}
+                <Heading
+                    elementType="{{ _headingElementType }}"
+                    size="xsmall"
+                    data-test-empty-tile="{{ _headingElementType }}"
+                >
+                    {{ _heading }}
+                </Heading>
+            {% endif %}
+            {% if _description %}
+                <Text color="secondary">{{ _description | raw }}</Text>
+            {% endif %}
         </div>
+    {% endif %}
+
+    {% if block('content') is not empty %}
```

Comment:
I think this block should be the starting one. E.g. children or anything else. And image, heading and description are now not part of this. I think they should.